wordpress-mobile/WordPress-Aztec-iOS
# Aztec: Format Bar

### Details:
In this PR we're matching Android's Format Bar behavior:

     **[ Scrollable Items ]** < **[ Fixed Items]**

We'll be adding new Format Bar actions in (upcoming) issues. Few notes:

- **Aztec.FormatBar** no longer inherits from **UIToolbar**.
- **Aztec.FormatBarItem** no longer inherits from **UIBarButtonItem**

Needs Review: @diegoreymendez @SergioEstevao 
Thanks in advance gentlemen!!

### Testing:
1. Launch Aztec on an iPhone SE device
2. Verify that the format bar animates, as soon as it becomes visible
3. Verify that the "Scrollable Items" work as expected
4. Rotate the device and verify everything looks right

Re-test on an iPhone 7 // 7 Plus // iPad (Multitasking). Note that whenever the horizontal sizeClass is regular (7 Plus in landscape, iPad fullscreen), the "Scrollable Items" spacing should be higher.

### Screenshots:
![simulator screen shot mar 10 2017 4 38 38 pm](https://cloud.githubusercontent.com/assets/1195260/23810513/b18aa0ac-05b0-11e7-949b-45d9c77367e5.png)

![simulator screen shot mar 10 2017 4 44 07 pm](https://cloud.githubusercontent.com/assets/1195260/23810549/cce12b32-05b0-11e7-9308-888315b96ae0.png)
